lib/Sema/ConstraintSystem.cpp

```diff
@@ -5996,6 +5996,53 @@ ValueDecl *ConstraintSystem::findResolvedMemberRef(ConstraintLocator *locator) {
   return choice.getDecl();
 }
 
+void SolutionApplicationTargetsKey::dump() const { dump(llvm::errs()); }
+
+void SolutionApplicationTargetsKey::dump(raw_ostream &OS) const {
+  switch (kind) {
+  case Kind::empty:
+    OS << "empty\n";
```

Comment:
Let's not add `\n` for these elements in case we'd ever want to dump key/value pairs somewhere just like we do for locators.

Reply:
Hm, I'm guessing some of these prints have `\n` in them, nevermind...